assignment3.txt

```diff
@@ -0,0 +1,7 @@
+1. it is invalid because a for loop has 3 requirements inside the for loop. (initialisation; condition; incrementat). It does not meet these requirements.
```

Comment:
It's true that a loop has those requirements, but if you assume that i is initialized to a number greater than 0, it'll work! There is another error - try and find it!